go/libraries/doltcore/sqle/diff_table.go

```diff
@@ -237,7 +252,21 @@ func (itr *diffRowItr) Next() (sql.Row, error) {
 		return nil, err
 	}
 
-	return doltRowToSqlRow(r, itr.sch)
+	sqlRow, err := doltRowToSqlRow(r, itr.sch)
+
+	if err != nil {
+		return nil, err
+	}
+
+	if hasTo && hasFrom {
+		sqlRow = append(sqlRow, "modified")
```

Comment:
You should pull out constants for these strings, maybe put them in the same package as the differ / joiner stuff.

Reply:
Moved to constants, but left in file.  I'm not sure I'd say this is how all diffs should label these fields in all cases.  It's just how I want them labeled for the diff_tables.